test/framework/yeb.py

```diff
@@ -94,6 +96,18 @@ def test_is_yeb_format(self):
         self.assertFalse(is_yeb_format(test_eb, None))
         self.assertFalse(is_yeb_format(None, raw_eb))
 
+
+    def test_join(self):
+        """ Test yaml_join function """
+
+        stream = "hw: !join [hello, world]"
```

Comment:
also use aliases/variables in the test for `!join`?

like you show in the example in the docs